src/Symfony/Component/Serializer/Tests/Normalizer/FormErrorNormalizerTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Tests\Normalizer;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\FormError;
+use Symfony\Component\Form\FormErrorIterator;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Serializer\Normalizer\FormErrorNormalizer;
+
+class FormErrorNormalizerTest extends TestCase
+{
+    /**
+     * @var FormErrorNormalizer
+     */
+    private $normalizer;
+
+    /**
+     * @var FormInterface
+     */
+    private $form;
+
+    protected function setUp(): void
+    {
+        $this->normalizer = new FormErrorNormalizer();
+
+        $this->form = $this->createMock(FormInterface::class);
+        $this->form->method('isSubmitted')->willReturn(true);
+        $this->form->method('all')->willReturn([]);
+
+        $this->form->method('getErrors')
+            ->willReturn(new FormErrorIterator($this->form, [
+                new FormError('a', 'b', ['c', 'd'], 5, 'f'),
+                new FormError(1, 2, [3, 4], 5, 6),
+            ])
+        );
+    }
+
+    public function testSupportsNormalizationWithWrongClass()
+    {
+        $this->assertFalse($this->normalizer->supportsNormalization(new \stdClass()));
+    }
+
+    public function testSupportsNormalizationWithNotSubmittedForm()
+    {
+        $form = $this->createMock(FormInterface::class);
+        $this->assertFalse($this->normalizer->supportsNormalization($form));
+    }
+
+    public function testSupportsNormalizationWithValidForm()
+    {
+        $this->assertTrue($this->normalizer->supportsNormalization($this->form));
+    }
+
+    public function testNormalize()
+    {
+        $expected = [
+            'code' => null,
+            'title' => 'Validation Failed',
+            'type' => 'https://symfony.com/errors/form',
+            'errors' => [
+                [
+                    'message' => 'a',
+                    'cause' => 'f',
+                ],
+                [
+                    'message' => '1',
+                    'cause' => 6,
+                ],
+            ],
+        ];
+
+        $this->assertEquals($expected, $this->normalizer->normalize($this->form));
+    }
+
+    // public function testNormalizeWithChildren()
```

Comment:
I think there is no need to leave the commented code here 😄

Reply:
I am working on this test because it fails. I commented this to see if other tests pass ✅